docs/INSTALL.md

```diff
@@ -0,0 +1,95 @@
+# Initial configuration of evercity node
+  you need:
+   - Linux server with 4 CPU Intel Xeon or AMD EPYC
+   - 16 Gb RAM
+   - 400 Gb free space on SSD
+
+# Build from GitHub on clean Ubuntu 18.04
+
+```bash
+# (!) In this document we work as user "evercity" with $HOME=/home/evercity and SUDO privilege
+```
+
+## Install required packages
+```bash
+# renew list of packages
+sudo apt update
+
+# install cmake, libssl, developer libraries, etc...
+sudo apt install -y cmake pkg-config libssl-dev git build-essential clang libclang-dev curl libz-dev
+```
+
+## Install Rust compiler and components for building node
+```bash
+# Download and install Rust
+sudo curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh # proceed with (Default) installation
+
+# "turn on" Rust environment in current shell
+source $HOME/.cargo/env
+
+# Update to lastest version of Rust
+rustup update
+
+## Install Rust nightly version(select needed "nightly" release
+
+```bash
+# set wanted nightly version of Rust
+# (!) I set version to 'nightly-2020-10-01' because with newer version (f.e. 'nightly-2021-02-13')
+# there is a compilation error
+NIGHTLY_RELEASE=nightly-2020-10-01
```

Comment:
it would be better if  rust toolchain version was specified only in one place - rust-toolchain file .

Reply:
fixed, all moved to rust-toolchain.toml